app/views/profiles/_edit.haml

```diff
@@ -9,10 +9,22 @@
   = render 'profiles/edit_public', :profile => profile, :aspect => aspect, :person => person
 
   %hr
+  :javascript
```

Comment:
If it doesn't matter fair enough. I just thought it was weird that `profile_is_public` doesn't have a matching element. I assume `profile[is_public]` becomes `id: profile_is_public`?